classes/class-twentytwenty-color.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * Color calculations.
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty
+ * @since 1.0.0
+ */
+
+if ( ! class_exists( 'TwentyTwenty_Color' ) ) {
+	/**
+	 * Color Calculations.
+	 */
+	class TwentyTwenty_Color {
+
+		/**
+		 * The HEX color.
+		 *
+		 * @access private
+		 * @since 1.0
+		 * @var string
+		 */
+		private $hex = '';
```

Comment:
Wht do you think about making this `protected` instead?

Reply:
I see no problem with making it `protected`, I'm just used to making `private` anything that should not be directly accessed ;)

I'll push a commit shortly